sql/moz-fx-data-shared-prod/firefox_desktop_derived/adclick_history_v1/query.sql

```diff
@@ -1,28 +1,20 @@
 WITH new_data AS (
   SELECT
     client_id,
-    sample_id,
```

Comment:
for my sake, any reason we're removing this?

Reply:
It makes it more efficient, trying to basically take out anything that isn't absolutely necessary to make the backfill run faster

Reply:
Basically, to clarify, if we leave it in, either I include it in the join with client ID (which slows stuff down since now it has to join on 2 things), or I do a max() or min() to only group by client ID, but still, that slows it down/adds extra work, when really we don't really need sample ID in this table, if that makes sense